xmtp_mls/src/groups/message_history.rs

```diff
@@ -1,11 +1,13 @@
 use xmtp_proto::{
     api_client::XmtpMlsClient,
     xmtp::mls::message_contents::plaintext_envelope::v2::MessageType::{
-        MessageHistoryRequest as HistoryRequest, MessageHistoryResponse as HistoryResponse,
+        Reply as HistoryResponse, Request as HistoryRequest,
```

Comment:
@tuddman I picked up this proto change of yours. I decided to take the path of least resistance and just alias everything back to the old names. Can I leave it to you to handle the final naming however you want in a subsequent PR.

Reply:
Sure.  I made the change in part so we'd not need to alias anything.

Reply:
They'll come in https://github.com/xmtp/libxmtp/pull/660